AssertionError: Event sender name must be specified

# Problem
I tried to use `one_call` method to fetch the forecast info for this loaction
lat: `55.761665`, lon: `37.606667` ( Moscow, RU )

I got this error message
```
  File "/Users/jeff/data3p/.venv/lib/python3.10/site-packages/pyowm/weatherapi25/weather_manager.py", line 532, in one_call
    return one_call.OneCall.from_dict(json_data)
  File "/Users/jeff/data3p/.venv/lib/python3.10/site-packages/pyowm/weatherapi25/one_call.py", line 100, in from_dict
    alerts = [NationalWeatherAlert.from_dict(item) for item in the_dict['alerts']]
  File "/Users/jeff/data3p/.venv/lib/python3.10/site-packages/pyowm/weatherapi25/one_call.py", line 100, in <listcomp>
    alerts = [NationalWeatherAlert.from_dict(item) for item in the_dict['alerts']]
  File "/Users/jeff/data3p/.venv/lib/python3.10/site-packages/pyowm/weatherapi25/national_weather_alert.py", line 105, in from_dict
    return NationalWeatherAlert(sender, title, description, start_time, end_time, tags)
  File "/Users/jeff/data3p/.venv/lib/python3.10/site-packages/pyowm/weatherapi25/national_weather_alert.py", line 33, in __init__
    assert sender, 'Event sender name must be specified'
AssertionError: Event sender name must be specified
```

# How to reproduce this bug
```python
mgr.one_call(lat=55.761665, lon=37.606667)
```


pyowm version: 3.3.0 ( latest )
python version: 3.10.6

--- a/pyowm/weatherapi25/national_weather_alert.py
+++ b/pyowm/weatherapi25/national_weather_alert.py
@@ -30,7 +30,7 @@ class NationalWeatherAlert:
     :raises: *ValueError* if any parameter has wrong type
     """
     def __init__(self, sender, title, description, start_time, end_time, tags=None):
-        assert sender, 'Event sender name must be specified'
+        assert sender is not None, 'Event sender name must be specified'
         assert title, 'Event title must be specified'
         assert description, 'Event description must be specified'
         assert start_time, 'Event start time must be specified'

